spatialmath/axisAngle.go

```diff
@@ -16,10 +16,10 @@ import (
 
 // R4AA represents an R4 axis angle.
 type R4AA struct {
-	Theta float64 `json:"th"`
-	RX    float64 `json:"x"`
-	RY    float64 `json:"y"`
-	RZ    float64 `json:"z"`
+	Theta float64 `json:"th_rads"`
+	RX    float64 `json:"x_mm"`
```

Comment:
RX, RY, and RZ are not mm and should not have a unit. There are unitless values.